src/nvim/terminal.c

```diff
@@ -359,6 +359,15 @@ void terminal_resize(Terminal *term, uint16_t width, uint16_t height)
     return;
   }
 
+  FOR_ALL_TAB_WINDOWS(tp, wp) {
```

Comment:
@mrbiggfoot this will loop through all tabs, is that intended? To loop only the current tab:

    FOR_ALL_WINDOWS_IN_TAB(wp, curtab)

Reply:
But the same terminal instance may be open in windows on multiple tabs, right? So yes, this is intended.